runtime/common/src/lib.rs

```diff
@@ -764,11 +765,26 @@ macro_rules! impl_config_traits {
             type ProposalBondMinimum = ProposalBondMinimum;
             type ProposalBondMaximum = ProposalBondMaximum;
             type RejectOrigin = EnsureRootOrTwoThirdsCouncil;
-            type SpendFunds = ();
+            type SpendFunds = Bounties;
             type SpendPeriod = SpendPeriod;
             type WeightInfo = weights::pallet_treasury::WeightInfo<Runtime>;
         }
 
+        impl pallet_bounties::Config for Runtime {
+            type BountyDepositBase = BountyDepositBase;
+            type BountyDepositPayoutDelay = BountyDepositPayoutDelay;
+            type BountyUpdatePeriod = BountyUpdatePeriod;
+            type CuratorDepositMultiplier = CuratorDepositMultiplier;
+            type CuratorDepositMax = CuratorDepositMax;
+            type CuratorDepositMin = CuratorDepositMin;
+            type BountyValueMinimum = BountyValueMinimum;
+            type DataDepositPerByte = DataDepositPerByte;
+            type Event = Event;
+            type MaximumReasonLength = MaximumReasonLength;
+            type WeightInfo = ();
+            type ChildBountyManager = ();
```

Comment:
Do you know what this is about?

Reply:
It allows creating sub tasks in bounty and such bounty is only completed when all of its sub tasks are completed.